modules/test/playwright/tests/osb-faro-web/page.spec.ts

```diff
@@ -58,6 +58,20 @@ const goToWithReferrer = async function (page, referrer, url) {
 	}, url);
 };
 
+const navigateToSitePage = async function (page, siteName, pageName) {
+	let pageNameURL = pageName.replace(/ /g, "-").toLowerCase();
+
+	if (siteName) {
+		let siteNameURL = siteName.replace(/ /g, "-").toLowerCase();
+
+		await page.goto(
+			`${liferayConfig.environment.baseUrl}/web/${siteNameURL}/` +
+				`${pageNameURL}`);
+	} else {
+		await page.goto(`${liferayConfig.environment.baseUrl}/${pageNameURL}`);
+	}
+};
```

Comment:
Hey @rachaelkoestartyo, could you move this function to the `analyticsSettings.ts` file? This is a file that we put the functions that will be more reusable on the DXP side